pennylane/devices/null_qubit.py

```diff
@@ -0,0 +1,173 @@
+            or iterable that contains unique labels for the subsystems as numbers (i.e., ``[-1, 0, 2]``)
+            or strings (``['auxiliary', 'q1', 'q2']``). Default 1 if not specified.
+    """
+
+    name = "Null qubit PennyLane plugin"
+    short_name = "null.qubit"
+    pennylane_requires = __version__
+    version = __version__
+    author = "Xanadu Inc."
+
+    def __init__(self, wires, *args, **kwargs):
+        defaultKwargs = {"shots": None}
+        kwargs = {**defaultKwargs, **kwargs}
+
+        self._operation_calls = defaultdict(int)
+        self._shots = kwargs["shots"]
+        self._shot_vector = None
+        self.custom_expand_fn = None
+        super().__init__(wires=wires, shots=self._shots)
+
+    # pylint: disable=arguments-differ
+    def apply(self, operations, *args, **kwargs):
+        for op in operations:
+            self._apply_operation(None, op)
+
+    def _apply_operation(self, state, operation):
+        if operation.__class__.__name__ in self._apply_ops:
+            return self._apply_ops[operation.base_name](state, axes=None, inverse=operation.inverse)
+
+        self._operation_calls[operation.name] += 1
+
+        wires = operation.wires
+        if operation in diagonal_in_z_basis:
+            return self._apply_diagonal_unitary(state, None, wires)
+        if len(wires) <= 2:
+            # Einsum is faster for small gates
+            return self._apply_unitary_einsum(state, None, wires)
+        return self._apply_unitary(state, None, wires)
+
+    def _apply_x(self, state, axes, **kwargs):
+        self._operation_calls["PauliX"] += 1
+
+    def _apply_y(self, state, axes, **kwargs):
+        self._operation_calls["PauliY"] += 1
+
+    def _apply_z(self, state, axes, **kwargs):
+        self._operation_calls["PauliZ"] += 1
+
+    def _apply_hadamard(self, state, axes, **kwargs):
+        self._operation_calls["Hadamard"] += 1
+
+    def _apply_s(self, state, axes, inverse=False):
+        self._operation_calls["S"] += 1
+
+    def _apply_t(self, state, axes, inverse=False):
+        self._operation_calls["T"] += 1
+
+    def _apply_sx(self, state, axes, inverse=False):
+        self._operation_calls["SX"] += 1
+
+    def _apply_cnot(self, state, axes, **kwargs):
+        self._operation_calls["CNOT"] += 1
+
+    def _apply_swap(self, state, axes, **kwargs):
+        self._operation_calls["SWAP"] += 1
+
+    def _apply_cz(self, state, axes, **kwargs):
+        self._operation_calls["CZ"] += 1
+
+    def _apply_toffoli(self, state, axes, **kwargs):
+        self._operation_calls["Toffoli"] += 1
+
+    def _apply_phase(self, state, axes, parameters, inverse=False):
+        pass
+
+    def expval(self, observable, shot_range=None, bin_size=None):
+        pass
+
+    def var(self, observable, shot_range=None, bin_size=None):
+        pass
+
+    @classmethod
+    def capabilities(cls):
+        capabilities = super().capabilities().copy()
+        capabilities.update(
+            model="qubit",
```

Comment:
Shouldn't it just inherit this property (and `supports_inverse_operations`, `supports_analytic_computation`, `return_state`) from default qubit?

Reply:
Good point, done!